CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# Release 2.35.0 22-03-2022
+
+- Update Envoy to v1.21.1 and Bazel to v5.0.0 (#670)
+- Remove envoy runtime flag preserve_downstream_scheme (#667)
+- Add option to allow discovery APIs (#665)
```

Comment:
You can remove this one, it is not meant for 3P

Reply:
Correct

Reply:
Done